### ISSUE TITLE
Create Namespace View

### ISSUE THREAD

**Issue by phildree** [amosproj/amos2023ss01-apache-pulsar-ui#39]
## User story
1. As a user
2. I need a namespace view
3. So that I can see relevant information about all namespaces in the current cluster

## Acceptance criteria
* [ ] Create the namespace view component that visualizes a list of namespaces according to the [mockup](https://github.com/amosproj/amos2023ss01-apache-pulsar-ui/blob/main/Documentation/amos_mockup.pdf) (for now these information should only be dummy data)
* [ ] Create a filter dropdown which provides a search box and checkboxes for all availables namespaces 
=> only selected cluster shall be displayed (default is all)
* [ ] TODO: Navigation

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
